src/Components/Crytpos.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable import/extensions */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { FadeLoader } from 'react-spinners';
+import { FetchCryptos } from '../Redux/features/CryptoSlice';
+import FormatNumber from '../Functions/FormatNumber';
+
+const Crytpos = () => {
+  const cryptosArr = [];
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const fetched = useSelector((state) => state.cryptoReducer);
+
+  useEffect(() => {
+    if (fetched.status === '') {
+      dispatch(FetchCryptos());
+    }
+  }, [fetched.status, dispatch]);
+
+  if (fetched.status === 'success') {
+    cryptosArr.push(fetched.cryptos[0]);
+  }
+
+  const handleClick = (cryptoObj) => {
+    navigate(
+      '/crypto-details',
+      {
+        state: {
+          cryptos: cryptoObj,
+        },
+      },
+
+    );
+  };
```

Comment:
- [ ] Nice job changing the URL as the previous reviewers suggested. However, you forgot to change the URL you navigate to when the user clicks on a "crypto" item which produced the following warning in the console. Please make sure to test your app with every change and see if everything works as intended.

![image](https://user-images.githubusercontent.com/62290545/206183202-a8831850-3a6a-4a1d-b95a-453940dfa4e0.png)

Reply:
## Hello @ShadyShawkat   👋🏼 
 Thanks for taking out time to review this PR. The required and optional changes have been duly noted and necessary changes will be implemented.